man/systemd.network.xml

```diff
@@ -1070,19 +1070,19 @@
         <varlistentry>
           <term><varname>InitialCongestionWindow=</varname></term>
           <listitem>
-            <para>The TCP initial congestion window or <literal>InitialCongestionWindow</literal>  is used during the start
-            of a TCP connection. During the start of a TCP session, when a client requests for a resource, the server's initial congestion window
-            determines how many data packets will be sent during the initial burst of data. Takes a number between between 1 and 4294967295 (2^32 - 1).
+            <para>The TCP <literal>InitialCongestionWindow</literal> is used during the start of a TCP connection. During the start
+            of a TCP session, when a client requests for a resource, the server's initial congestion window determines how many data bytes
+            will be sent during the initial burst of data. Takes a number between between 1 and 4294967295 (2^32 - 1).
```

Comment:
The last sentence should mention the suffix. E.g., "Takes a size in bytes between 1 and 4294967295 (2^32 - 1). The usual suffixes K, M, G are supported and are understood to the base of 1024."

Reply:
Also, `0` means not using the feature, right? If so, please I think it is worth to mention that.

Reply:
No kernel just sets to TCP_INIT_CWND           10